### PR TITLE
Standardize height prop

### PR DESCRIPTION
**Pull request by Matt-Butler**
### Summary
Updating the height PropType of image and progress bar to promote consistency.

@cerner/terra

https://github.com/cerner/terra-core/issues/225

### CODE REVIEW

File: packages/terra-progress-bar/src/ProgressBar.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const propTypes = {
   /**
    * Sets the size of the progress-bar from the following values; `tiny`, `small`, `medium`, `large` and `huge`
    */
-  height: PropTypes.oneOf(['tiny', 'small', 'medium', 'large', 'huge']),
+  progressbarHeight: PropTypes.oneOf(['tiny', 'small', 'medium', 'large', 'huge']),
```
Comment (bjankord) [inline packages/terra-progress-bar/src/ProgressBar.jsx]: Lets update this prop name to `heightSize`

Comment (Matt-Butler) [inline packages/terra-progress-bar/src/ProgressBar.jsx]: +1 I agree
https://github.com/cerner/terra-core/pull/247/commits/3000669a7d9045d4624df03a05e91803ac0cff14

**Comment by mjhenkes** [comment]
Correct me if i'm wrong, but this would be major version bump for image and progress bar? Are we saying that pre 1.0 versions don't need to be passive?

**Comment by Matt-Butler** [comment]
> Correct me if i'm wrong, but this would be major version bump for image and progress bar? Are we saying that pre 1.0 versions don't need to be passive?
At this stage I'm ok either way, I just want to be able to state our strategy.


My 2 cents, anything < 1.0 has the possibility to be non-passive. I would do a minor release for progress bar and image.

**Pull request merged**